Kudu.Core/Jobs/ContinuousJobsManager.cs

```diff
@@ -18,6 +18,19 @@ public class ContinuousJobsManager : JobsManagerBase<ContinuousJob>, IContinuous
     {
         private const string StatusFilesSearchPattern = ContinuousJobStatus.FileNamePrefix + "*";
         private const string Localhost = "127.0.0.1";
+        public bool ListJobRefreshNeeded
```

Comment:
Also, why not just set the cache to null when we need a refresh and do away with this separate prop?